src/app/title/page.tsx

```diff
@@ -102,6 +104,58 @@ const DependencyTabPosterContainer = styled.div`
 	gap: 0.5rem;
 `;
 
+const SequentialOrderBlock = styled.div`
+	align-items: center;
+	display: flex;
+	flex-direction: row;
+	gap: 8rem;
+	height: 40rem;
+	max-width: 150rem;
+`;
+
+const SequentialItem = styled.div`
+	align-items: center;
+	display: flex;
+	height: 100%;
+	justify-content: center;
+	flex-direction: column;
+	gap: 1rem;
+
+	img {
+		max-height: 100%;
+		object-fit: contain;
+	}
+
+	h2 {
+		font-size: 2rem;
+		font-weight: bold;
+		opacity: 0.8;
+	}
+
+	h3 {
+		font-size: 1.5rem;
+		font-weight: normal;
+		opacity: 0.6;
+	}
+`;
+
+const SequentialAdjacent = styled(SequentialItem).withConfig({
+	shouldForwardProp: (prop) => prop !== "alignRight",
+})<{ alignRight?: boolean }>`
```

Comment:
I'm pretty sure you can simplify this by prefixing the props with `$`
```suggestion
const SequentialAdjacent = styled(SequentialItem)<{ $alignRight?: boolean }>`
```

Reply:
As a sidenote, this alignment doesn't seem to do anything for me (Tested on Chrome)

Reply:
💀 oh

it seems to still work visually so i'll just leave it as is for now